README.md

```diff
@@ -10,30 +10,51 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator
 
-### SFC MachineConfig
+### SFC out of tree driver
 
-To apply the SFC MachineConfig, it is a prerequisite that there exists a local docker registry containing an image with the SFC module built and present in /opt in the node filesystem. This is expected to be present at the following location:
+The recommended way of installing onload's sfc driver is using [KMM](https://github.com/rh-ecosystem-edge/kernel-module-management/)
```

Comment:
This should link to [official Redhat documentation](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html) and/or [Kubernetes KMM documentation](https://kmm.sigs.k8s.io/), not the bleeding edge codebase.